frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts

```diff
@@ -67,6 +67,12 @@ const MiniFilters: SharedListMiniFilter[] = [
         name: 'Autocapture',
         tooltip: 'Autocapture events such as clicks and inputs',
     },
+    {
+        tab: SessionRecordingPlayerTab.EVENTS,
+        key: 'events-exceptions',
+        name: 'Exceptions',
+        tooltip: 'Exception events from PostHog or its Sentry integration',
+    },
```

Comment:
I like this _but_ I worry that adding it without having our own exception tracking will confuse people... 
I guess we can see if anyone clicks it and then maybe think about adding a helpful empty state like "You're project has no exception events - here is what you can do to enable them"

Reply:
Ah, good call!